### ISSUE TITLE
Outputs `import template from './foo.js'`

### ISSUE THREAD

**Issue by imjared** [CVarisco/create-component-app#8]
Given input:

> 1. What kind of components do you want to create ? stateless
> 2. What is the name of the component ? Foo
> 3. Do you want to connect the component to redux ? Yes
> 4. Do you want an index file on your folder ? Yes
> 5. What kind of extension do you use for js files ? js
> 6. What kind of extension do you use for css files ? css
> 7. Where do you want create your component ? ./src/components

Output index.js:
```
import template from './Foo.js'
import { connect } from 'react-redux';

const mapDispatchToProps = (dispatch, ownProps) => {
    return {}
}

const mapStateToProps = (state, ownProps) => {
    return {};
};

const Foo = connect(
    mapStateToProps,
    mapDispatchToProps,
)(Foo);

export default Foo
```

Should `connect` connect `template` instead? Am I missing something?

**Comment by CVarisco** [comment]
Hi @imjared,
Thanks a lot for your issue 🙂 
You are right, I think that the first line needs to be `import Foo from './Foo.js'`

What do you think about it ?

**Comment by CVarisco** [comment]
So the new template index needs to be like this:
```javascript
import Foo from './Foo.js'
import { connect } from 'react-redux';

const mapDispatchToProps = (dispatch, ownProps) => {
    return {}
}

const mapStateToProps = (state, ownProps) => {
    return {};
};

export default connect(
    mapStateToProps,
    mapDispatchToProps,
)(Foo);
```

It is ok for you ? Correct ? 🙂

**Comment by imjared** [comment]
I think that'd do it!

**Comment by CVarisco** [comment]
Hi @imjared 
I published a new version (v0.4.1) with the fixes:
fcb9c6ed6f90cb4a892d1c0aedd3451978ab6cc2
c2433f85cc7c5e82ce3c601360d5529b3cdf7dcf

Could you check if for you is ok ? 
Thanks a lot!
